### ISSUE TITLE
Tests in `launcher_parse_args_test.py` fail in VSCode devcontainer

### ISSUE THREAD

**Issue by motus** [microsoft/MLOS#575]
...but only when launched from VSCode environment. They pass when running pytest in terminal. The following assertion fails in `launcher_parse_args_test.py::test_launcher_args_parse_1`:
```python
>       assert path_join(launcher.global_config["pathVarWithEnvVarRef"], abs_path=True) \
            == path_join(expected_root_pwd, "foo", abs_path=True)
E       AssertionError: assert '/workspaces/MlosCore/foo' == '/vscode/vscode-server/bin/linux-x64/d037ac076cee195194f93ce6fe2bdfe2969cc82d/foo'
E         - /vscode/vscode-server/bin/linux-x64/d037ac076cee195194f93ce6fe2bdfe2969cc82d/foo
E         + /workspaces/MlosCore/foo
```

**Comment by bpkroth** [comment]
I think I fixed that already in #553 no?

**Comment by bpkroth** [comment]
Per discussions elsewhere, things seem to be working for me with latest vscode and latest docker for #570.
Let's make sure all of those things are handled first and then I'll can come back to this if necessary, but I'd like to shift energy back towards other efforts if this isn't a major blocker.  Make sense?
